src/main/java/commonMethods/CommonMethods.java

```diff
@@ -1,4 +1,4 @@
-package AbstractComponents;
+package commonMethods;
```

Comment:
package names are supposed to be all in lower case `commonmethods` or you can have `common.methods`

Reply:
ooh, I thought if you are using two words anywhere in Java, you need to use a capital letter to split it